test/python/visualization/test_gate_map.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A test for visualizing device coupling maps"""
+import unittest
+import os
+import matplotlib.pyplot as plt
+from qiskit import IBMQ
+from qiskit.test import QiskitTestCase
+from qiskit.visualization.gate_map import _GraphDist, plot_gate_map
+from .visualization import path_to_diagram_reference, QiskitVisualizationTestCase
+
+
+class TestGateMap(QiskitVisualizationTestCase):
+    """ visual tests for plot_gate_map """
+    backend = []
+
+    def setup(self):
+        """ setup for backend """
+        self.backend = IBMQ.backends(filters=lambda x: not x.configuration().simulator)
```

Comment:
`self.backend` -> `self.backends` as it is a list of backends, not a singular device.

Reply:
Ah my bad. Have changed this as well. 
Thank you again for your feedback and advice on this PR!